packages/x-date-pickers/src/internals/components/PickersProvider.tsx

```diff
@@ -45,4 +45,8 @@ export interface PickersContextValue {
    * `true` if the picker is open, `false` otherwise.
    */
   open: boolean;
+  /**
+   * The ownerState of the picker.
+   */
+  ownerState: PickerOwnerState<TValue>;
```

Comment:
Slots handled by parts of the picker (fields, views, layout, etc...) need to have access to the ownerState to pass it to their own slots.

Do you think the `ownerState` should be in the public context (like I did here) or in a standalone private context?

Reply:
We decided to move it to a private context